packages/web-components/src/tabs/tabs.styles.ts

```diff
@@ -0,0 +1,250 @@
+    transition-property: transform;
+    transition-duration: ${durationSlow};
+    transition-timing-function: ${curveDecelerateMax};
+  }
+
+  :host([size='small']) ::slotted(fluent-tab) {
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    padding-top: ${spacingVerticalSNudge};
+    padding-bottom: ${spacingVerticalSNudge};
+    padding-left: ${spacingHorizontalSNudge};
+    padding-right: ${spacingHorizontalSNudge};
+    border-radius: ${borderRadiusMedium};
+  }
+
+  :host([size='large']) ::slotted(fluent-tab) {
+    font-size: ${fontSizeBase400};
+    line-height: ${lineHeightBase400};
+    padding-top: ${spacingVerticalL};
+    padding-bottom: ${spacingVerticalL};
+    padding-left: ${spacingHorizontalMNudge};
+    padding-right: ${spacingHorizontalMNudge};
+    border-radius: ${borderRadiusMedium};
+  }
+
+  /* ::after adds the active indicator  */
+  :host([orientation='horizontal']) ::slotted(fluent-tab)::after {
+    height: ${strokeWidthThicker};
+    margin-top: auto;
+    transform-origin: left;
+    transform: translateX(var(${TabTokenNames.tabIndicatorOffset})) scaleX(var(${TabTokenNames.tabIndicatorScale}));
+  }
+  /* ::before adds a secondary indicator placeholder that appears right after click on the active tab */
+  :host ::slotted(fluent-tab)::before {
+    height: ${strokeWidthThicker};
+    border-radius: ${borderRadiusCircular};
+    content: '';
+    inset: 0;
+    position: absolute;
+    margin-top: auto;
+  }
+
+  :host([orientation='horizontal']) ::slotted(fluent-tab[aria-selected='false']:hover)::after {
+    height: ${strokeWidthThicker};
+    margin-top: auto;
+    transform-origin: left;
+  }
+
+  :host ::slotted(fluent-tab[aria-selected='true'])::before {
+    background-color: ${colorNeutralForegroundDisabled};
+  }
+  :host([orientation='vertical']) ::slotted(fluent-tab)::before {
+    height: unset;
+    width: ${strokeWidthThicker};
+    margin-inline-end: auto;
+    transform-origin: top;
+  }
+
+  :host([orientation='horizontal'][size='small']) ::slotted(fluent-tab)::after,
+  :host([orientation='horizontal'][size='small']) ::slotted(fluent-tab)::before,
+  :host([orientation='horizontal'][size='small']) ::slotted(fluent-tab:hover)::after {
+    right: ${spacingHorizontalSNudge};
+    left: ${spacingHorizontalSNudge};
+  }
+  :host([orientation='horizontal']) ::slotted(fluent-tab)::after,
+  :host([orientation='horizontal']) ::slotted(fluent-tab)::before,
+  :host([orientation='horizontal']) ::slotted(fluent-tab:hover)::after {
+    right: ${spacingHorizontalMNudge};
+    left: ${spacingHorizontalMNudge};
+  }
+  :host([orientation='horizontal'][size='large']) ::slotted(fluent-tab)::after,
+  :host([orientation='horizontal'][size='large']) ::slotted(fluent-tab)::before,
+  :host([orientation='horizontal'][size='large']) ::slotted(fluent-tab:hover)::after {
+    right: ${spacingHorizontalMNudge};
+    left: ${spacingHorizontalMNudge};
+  }
```

Comment:
Horizontal is the default here, can we reduce the complexity of these selectors?